src/main/python/image_processing/sample_extract.py

```diff
@@ -25,6 +38,9 @@ def extract(self, img):
             id_details: JSON obj (The extracted information)
         Todo:
         """
+        if 'remove_face' in self.preferences:
+            remove_face = self.preferences['remove_face'] == 'true'
```

Comment:
When posted by AJAX the value is true or false (lowercase) and Python uses True or False (uppercase). If you convert the received 'true' to 'True' it is seen as a string and does not work, hence the comparison.